pdb-slave/MARCH4-PDB/lib/PDBI2C/HVControl.cpp

```diff
@@ -143,4 +143,25 @@ void HVControl::setAllHV(uint8_t code){
         }
     }
     this->write();
+}
+
+void HVControl::setAllHVStagedStartup(uint8_t code){
+    // Loop through all HVOn pins and turn on HV depending on code
+    // Staged startup is meant to turn on all HV nets seperately, to minimize inrush currents
+    // Debug: turn on LED 4 if staged startup is called
+    DigitalOut debugLED(LED4, false);
```

Comment:
I like the "debug led" as well and I would love to see it back as a feature. So maybe change the name if you implement this as a feature?

Reply:
Will do this